.github/workflows/nodejs.yml

```diff
@@ -28,6 +28,7 @@ jobs:
     - run: npm ci
     - run: npm run lint && npm run solhint
     - run: npm run generate
+    - run: npm run tsc
```

Comment:
It didn't look like we were running typescript transpiler in CI so added this. If this is being done somewhere else can remove this step.

Reply:
Not needed, remove